docs/develop/go/temporal-nexus.mdx

```diff
@@ -45,17 +49,13 @@ Prerequisites:
 - [Install the latest Temporal Go SDK](https://docs.temporal.io/develop/go/core-application#install-a-temporal-sdk)
   (v1.33.0 or higher recommended)
 
-### Start the Temporal Development Server {#start-the-temporal-development-server}
-
-Start the Temporal Development Server.
-
 ```
 temporal server start-dev
 ```
 
-This command automatically starts the Web UI, creates the default Namespace, and uses an in-memory database.
+This command automatically starts the Temporal development server with the Web UI, and creates the `default` Namespace. It uses an in-memory database, so do not use it for real use cases.
 
-The Temporal Server should be available on `localhost:7233` and the Temporal Web UI should be accessible at [http://localhost:8233](http://localhost:8233).
+The Temporal Web UI should now be accessible at [http://localhost:8233](http://localhost:8233), and the Temporal Server should now be available for client connections on `localhost:7233`.
```

Comment:
Reordered slightly: humans basically click on web links (rather than issue gRPC to TCP sockets!)